distributor/src/distributor/save.bal

```diff
@@ -37,6 +37,14 @@ const UPDATE_CALLBACK = "UPDATE callbacks SET callback = ? WHERE username = ?";
 const SELECT_CALLBACKS = "SELECT * FROM callbacks";
 const DROP_CALLBACKS_TABLE = "DROP TABLE callbacks";
 
+const string CREATE_RECIPIENT_TABLE = "CREATE TABLE IF NOT EXISTS smsRecipients (" +
```

Comment:
Can we also store the user name that's registering the mobile # here? That will be useful for debug purposes.